plugins/com.google.cloud.tools.eclipse.util.test/testData/templates/appengine/loggingProperties.txt

```diff
@@ -0,0 +1,13 @@
+# A default java.util.logging configuration.
```

Comment:
These comments aren't necessary except for line 12

Reply:
Just have a pointer to https://cloud.google.com/appengine/docs/standard/java/logs/?